src/applications/model/crazytaxi-streaming-server.cc

```diff
@@ -29,95 +29,102 @@
+
+TypeId
+CrazyTaxiStreamingServer::GetTypeId (void)
+{
+  static TypeId tid = TypeId ("ns3::CrazyTaxiStreamingServer")
+      .SetParent<GamingStreamingServer> ()
+      .SetGroupName("Applications")
+      .AddConstructor<CrazyTaxiStreamingServer> ()
+  ;
+  return tid;
+}
+
+CrazyTaxiStreamingServer::CrazyTaxiStreamingServer ()
+  : GamingStreamingServer ()
+{
+  NS_LOG_FUNCTION (this);
+  InitializeStreams ();
+};
+
+CrazyTaxiStreamingServer::CrazyTaxiStreamingServer (Ipv4Address ip, uint16_t port)
+  : GamingStreamingServer (ip, port)
+{
+  NS_LOG_FUNCTION (this);
+  InitializeStreams ();
+}
+
+CrazyTaxiStreamingServer::~CrazyTaxiStreamingServer ()
+{
+  NS_LOG_FUNCTION (this);
+}
+
+void
+CrazyTaxiStreamingServer::InitializeStreams ()
+{
+  /** Add CBR audio stream */
+  Ptr<ConstantRandomVariable> pktCbrAudio = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (216));
+  Ptr<ConstantRandomVariable> iatCbrAudio = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (10));
+  AddNewTraffic (pktCbrAudio, iatCbrAudio);
+
+  /** Add Cursor stream */
+  Ptr<ConstantRandomVariable> pktCursor = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (28));
+  Ptr<ConstantRandomVariable> iatCursor = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (50));
+  AddNewTraffic (pktCursor, iatCursor);
+
+  /** Add VBR audio stream */
+  // Packet size
+  Ptr<ConstantRandomVariable> pktVbrAudio1 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (244));
+  Ptr<ConstantRandomVariable> pktVbrAudio2 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (1384));
+
+  Ptr<MixtureRandomVariable> pktVbrAudio = CreateObject<MixtureRandomVariable> ();
+  pktVbrAudio->SetRandomVariables ({pktVbrAudio1, pktVbrAudio2}, {0.0776, 0.9224});
+
+  // Packet inter-arrival time
+  Ptr<ConstantRandomVariable> iatVbrAudio = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (50));
+  AddNewTraffic (pktVbrAudio, iatVbrAudio);
+
+  /** Add video stream */
+  // Packet size
+  Ptr<UniformRandomVariable> pktVideo1 = CreateObjectWithAttributes<UniformRandomVariable> ("Min", DoubleValue (0),
+                                                                                            "Max", DoubleValue (1355));
+  Ptr<ConstantRandomVariable> pktVideo2 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (1356));
+
+  Ptr<MixtureRandomVariable> pktVideo = CreateObject<MixtureRandomVariable> ();
+  pktVideo->SetRandomVariables ({pktVideo1, pktVideo2}, {0.3606, 0.6394});
+
+  // Packet inter-arrival time
+  Ptr<ConstantRandomVariable> iatVideo1 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue (0));
+  Ptr<LogNormalRandomVariable> iatVideo2 = CreateObjectWithAttributes<LogNormalRandomVariable> ("Mu", DoubleValue (0.34),
```

Comment:
again, this should be a 3 params log normal. please add a TODO in order to not forget about it